### ISSUE TITLE
Can't get Gotek 435 into programming mode.

### ISSUE THREAD

**Issue by Deathstryker** [keirf/flashfloppy#876]
Hello, I feel like I have pretty decent experience with stuff like this but for some reason I can't get my Gotek into programming mode. This is actually the second one I've tried. The first one had a 415 chip set and I ended up returning it. But I've downloaded the ISP drivers and programmer and installed the driver and restarted when it prompted me. As you can see in the image I have the pins for programming soldered. But when I plug it into my laptop, I get nothing. Nothing shows up in Device Manager and nothing shows in the programmer. I've tried one brand new USB A-A cable and 3 other USB A-C cables. Also tried on another laptop with the same result. Any idea what I'm doing wrong?

![PXL_20240302_001026770](https://github.com/keirf/flashfloppy/assets/25914361/4a7551b1-c277-4245-bda8-f2a8d5007670)

**Comment by Deathstryker** [comment]
Sorry, I just figured it out. I did some continuity tests on the cable and found out one of the pins wasn't making contact with the gotek. I pulled the cable out just a bit because apparently it can be pushed in too far and then my laptop read it easily. I'm not sure if it was the cable's fault or the USB ports they use in these Goteks (maybe a combination of both) but just figured I'd at least post my solution for others that may be having this issue.
